locales/lt-LT/messages.po

```diff
@@ -0,0 +1,1833 @@
+msgstr "Meta duomenų eksportavimui reikalinga core_kernel_classes_Resource instancija, kad įterpti meta duomenis"
+
+msgid "Metadata import requires an instance of DomManifest to extract metadata"
+msgstr "Meta duomenų eksportavimui reikalinga DomManifest instancija, kad įterpti meta duomenis"
+
+msgid "Microsoft Excel"
+msgstr "Microsoft Excel"
+
+msgid "Microsoft Powerpoint"
+msgstr "Microsoft Powerpoint"
+
+msgid "Microsoft Word"
+msgstr "Microsoft Word"
+
+msgid "MIME-type"
+msgstr "MIME tipas"
+
+msgid "Min"
+msgstr "Min."
+
+msgid "Minimal  score for this interaction."
+msgstr "Minimalūs balai šiai užduočiai."
+
+msgid "Modal feedback is shown to the candidate directly following response processing."
+msgstr "Modalinis atsiliepimas yra rodomas kandidatui tiesiogiai sekant atsakymo apdorojimą."
+
+msgid "Modal Feedback Prop."
+msgstr ""
+
+msgid "Modal Feedbacks"
+msgstr "Modaliniai Atsiliepimai"
+
+msgid "MP4 video"
+msgstr "MP4 vaizdo įrašas"
+
+msgid "MPEG audio"
+msgstr "MPEG garso įrašas"
+
+msgid "MPEG video"
+msgstr "MPEG vaizdo įrašas"
+
+msgid "Multiply"
+msgstr "Dauginti"
+
+msgid "negative"
+msgstr "negatyvus"
+
+#, tao-public
+msgid "New item"
+msgstr "Naujas elementas"
+
+msgid "No"
+msgstr "Ne"
+
+msgid "no configuration property needed"
+msgstr "nereikalaujama konfigūracijos savybė"
+
+msgid "No file selected"
+msgstr "Joks failas nepasirinktas"
+
+msgid "no item content"
+msgstr "nėra elemento turinio"
+
+msgid "No Items could be imported from the given IMS QTI package."
+msgstr "Joks Elementas negali būti importuotas iš pateikto IMS QTI paketo."
+
+msgid "No preview available"
+msgstr "Nėra galimos peržiūros"
+
+msgid "No value set"
+msgstr "Nenustatyta jokia reikšmė"
+
+msgid "None"
+msgstr "Nėra"
+
+msgid "none"
+msgstr "nėra"
+
+msgid "None defined yet."
+msgstr "Niekas nenustatyta."
+
+msgid "not a QTI item"
+msgstr "tai ne QTI elementas"
+
+msgid "Number of associations"
+msgstr "Ryšių kiekis"
+
+msgid "of"
+msgstr "iš"
+
+msgid "OK"
+msgstr "Gerai"
+
+msgid "one association pair"
+msgstr "vienas ryšys per porą"
+
+msgid "One or more QTI components are not supported by the system. The system returned the following error: %s\n"
+""
+msgstr "Vienas ar daugiau QTI komponentų nėra palaikomas sistemos. Sistema gražino šią klaidą: %s\n"
+""
+
+msgid "Order"
+msgstr "Eiliškumas"
+
+msgid "Order Interaction"
+msgstr "Eiliškumo Užduotis"
+
+msgid "Order the areas (hotspots) displayed on a picture."
+msgstr ""
+
+msgid "Orientation"
+msgstr "Orientacija"
+
+msgid "Outcome Declarations"
+msgstr ""
+
+msgid "Pair scoring"
+msgstr "Poros vertinimas"
+
+msgid "Pattern"
+msgstr "Šablonas"
+
+msgid "Pause"
+msgstr "Pristabdyti"
+
+msgid "PDF file"
+msgstr "PDF failas"
+
+msgid "Pi"
+msgstr "Pi"
+
+msgid "Plain text"
+msgstr "Paprastas tekstas"
+
+msgid "Please add one or more gaps to the text in question mode."
+msgstr ""
+
+msgid "Please create areas that correspond to the response and associate them a score.\n"
+""
```

Comment:
Is this ok? It looks like it's missing smthing.

Reply:
yes. it is normal

Reply:
i asked Anshul